fastanime/cli/interfaces/anilist_interfaces.py

```diff
@@ -352,14 +352,17 @@ def fetch_streams(config: "Config", anilist_config: QueryDict):
     )
     # -- update anilist info if user --
     if config.user and episode_number:
-        AniList.update_anime_list(
-            {
-                "mediaId": anime_id,
```

Comment:
Restricting the update of an episode on whether the episode was **potentially** already watched is problematic. Since it limits the freedom of a user who for example doesnt remember where they left of from another platform they used to watched the same episode on. So am not going to merge this change.

Reply:
I understand your argument, but this is something I will still put in my local copy of the app. It is annoying to me that my progress is set back to an earlier episode just because I played it to check something, grab screenshots etc.